### DISCUSSION TITLE
How to run Amiga OS4 (Sam460) on Apple Silicon

### DISCUSSION THREAD

**Discussion by ikir83** [utmapp/UTM#5671] [UTM Help (macOS)]
Hi everybody. Does anyone know how to emulate AmigaOS4 on Apple Silicon?

I've the original Sam460 iso, but I can only get to (SLB) error, and I need to wait a lot before anything to appear other than "Guest has not initialized the display (yet)

UTM conf:
ACube Sam460
512MB RAM
SCSI disk
Bosch-VGA

**Comment by tifasoftware** [comment]
Use Video Card sm501, its implemented in UTM
https://amiga-news.de/en/news/AN-2023-03-00113-EN.html

**Reply by ikir83** [reply]
Great, yesterday I've understood that the virtual machines is not getting keyboard input. Changed to sm501 thanks

**Comment by ikir83** [comment]
Anyone got it working?

**Reply by vgenop** [reply]
Never worked on UTM for me. I had to run qemu directly from terminal and Amiga OS4 worked fine

**Comment by ikir83** [comment]
Any other experience to share about it? I've tried again, VGA seems to be supported now but still can't get keyboard to work and it seems IDE/SATA is not well supported.
